README.md

```diff
@@ -38,12 +38,13 @@ $ # Add the content of the example configuration file below into ./inc/base/conf
 $ chmod 0777 ./inc/base/config.php
 $ chmod -R 0777 ./ext_inc/
 $ docker-compose up
+$ docker-compose run php composer install
```

Comment:
Isn't this triggered by the Dockerfile already?

Reply:
Yes, but depending on when the upstream image was last pushed, that might be outdated by now.
Also, when the `composer.json` gets modified, a rerun might also be needed.
Therefore, to avoid unexpected side-effects it's better to add it to the docs, in my opinion.